catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -181,6 +192,29 @@ public Metacard getMetacard(String id)
             }
             resultTypes.put(metacardType.getName(), attributes);
         }
+        for (InjectableAttribute attribute : injectableAttributes) {
```

Comment:
one iterates over the metacard type -> then each attribute in the type
the other iterates over each Injectable attribute -> then over each type that injectable attribute belongs in